### ISSUE TITLE
[BUG] [SP-API] [Issue] createInboundPlan requires phoneNumber

### ISSUE THREAD

**Issue by MojoJake** [amzn/selling-partner-api-models#654]
## I'm submitting a...

<pre><code>[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## SP-API Name and Version or Report/Feed Type
/inbound/fba/2024-03-20/inboundPlans/createInboundPlan

## Expected Behavior

createInboundPlan didn't used to require a phone number as part of the sourceAddress. The documentation doesn't mention phoneNumber at all for the "sourceAddress" https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-reference#createinboundplanrequest

## Current Behavior

createInboundPlan fails whether I provide a phone number or not. It returns: 
_"1 validation error detected: Value null at 'request.sourceAddress.phoneNumber' failed to satisfy constraint: Member must not be null"_

## Your Environment

I am using the node npm package.

**Comment by or-en** [comment]
This issue has returned

[400] 1 validation error detected: Value null at 'request.sourceAddress.phoneNumber' failed to satisfy constraint: Member must not be null (InvalidInput)

Anyone had it as well?

**Comment by 111L12** [comment]
[400] 1 validation error detected: Value null at 'request.sourceAddress.phoneNumber' failed to satisfy constraint: Member must not be null (InvalidInput)

**Comment by 111L12** [comment]
[400] 1 validation error detected: Value null at 'request.sourceAddress.phoneNumber' failed to satisfy constraint: Member must not be null (InvalidInput)

**Comment by 111L12** [comment]
> 此问题又回来了
> 
> [400] 检测到 1 个验证错误：“request.sourceAddress.phoneNumber”处的值为 null 无法满足约束：成员不得为 null （InvalidInput）
> 
> 有人也有吗？

I also encountered this problem today. Have you solved it?

**Comment by miraemor** [comment]
Still experiencing validation error. Neither of requests is valid (with or without "phoneNumber" in sourceAddress section)

**Comment by shreeharsh-a** [comment]
Please reach out to [developer support](https://developer.amazonservices.com/support) so that we can better help you with issues related to SP API.

`Note: SP API or docs related issues / troubleshooting support is managed by a different team. We can help if the issue is related to the content published on this repo.`

**Comment by shantikolli** [comment]
@shreeharsh-a Many a times I have reached to developer support, created cases but there is no proper information provided. I have created one case which is with the support team from almost 5 days, yet there is no update. Any issue is raised, all they are doing is to provide the documentation link which is not useful as every one working on SP-API API endpoints are already aware of the documentation and following the same.
